src/config/syscheck-config.c

```diff
@@ -1212,7 +1210,7 @@ static void parse_synchronization(syscheck_config * syscheck, XML_NODE node) {
             if (t == -1) {
                 mwarn(XML_VALUEERR, node[i]->element, node[i]->content);
             } else {
-                syscheck->sync_response_timeout = t;
+                mwarn("This configuration variable has been deprecated. It will have no effect.");
```

Comment:
Some suggestions here:
- Let's use a debug1 log to avoid a warning by default when upgrading
- Let's include the variable name in the log. That way, the user can find what setting is deprecated in his configuration
- Let's avoid processing the read value if we are not working with it
```suggestion
    } else if (strcmp(node[i]->element, xml_response_timeout) == 0) {
        mdebug1("'%s' has been deprecated. This setting is skipped.", xml_response_timeout);
    }
```

Reply:
done